packages/@sanity/base/src/change-indicators/ChangeBar.styled.tsx

```diff
@@ -0,0 +1,211 @@
+import React from 'react'
+import styled, {css} from 'styled-components'
+import {Theme} from '@sanity/ui'
+import {EditIcon} from '@sanity/icons'
+
+interface BarAndShapeInterface {
```

Comment:
The "*Interface" suffix seems a bit redundant. Since it declares the given context variables maybe it could be named "ThemeContext" or something?

Reply:
Agreed, changing it 👍